frontend/pages/index.jsx

```diff
@@ -119,7 +119,7 @@ export async function getStaticProps(context) {
             batches
           }
         },
-        revalidate: 1
+        unstable_revalidate: 1
```

Comment:
why have you changed it to unstable_revalidate?
we upgraded to next 9.5.1 so it's only revalidate now